roles/nexus/templates/nexususer.yml.j2

```diff
@@ -0,0 +1,13 @@
+{% for user in rhsso_users|json_query("[?(@.jenkinsRole)]") %}
```

Comment:
Should filter on `nexusRole`, not `jenkinsRole`

Reply:
You are correct, fixed.

Reply:
We can remove the oauth_admin_users. I chose not to at this time because we didn't have the devsecops api updated to actually handle roles properly.

Reply:
I think it makes sense so that we're not attempting to create users twice during this role